serverless.yml

```diff
@@ -61,6 +61,6 @@ functions:
     events:
       # cron(Minutes Hours Day-of-month Month Day-of-week Year)
       # Either `day-of-month` or `day-of-week` must be a question mark (?)
-       - schedule: cron(30 12,14,18,20 ? * MON,TUE,WED,THU,FRI *)
+       - schedule: cron(30 0/2 ? * MON,TUE,WED,THU,FRI *)
```

Comment:
Do you want to change `MON,TUE,WED,THU,FRI` to `*` too?

Reply:
Yeah, probably not a bad idea, even though alerts may go completely unnoticed during that time.